megamek/src/megamek/client/ui/swing/AccessibilityWindow.java

```diff
@@ -182,7 +181,7 @@ private void processAccessibleGUI() {
             // Cursor over the hex.
             gui.bv.mouseAction(selectedTarget, 3, InputEvent.BUTTON1_DOWN_MASK, MouseEvent.BUTTON1);
             // Click.
-            ((BoardView1) gui.getBoardView()).mouseAction(selectedTarget, 1, InputEvent.BUTTON1_DOWN_MASK, MouseEvent.BUTTON1);
+            ((BoardView) gui.getBoardView()).mouseAction(selectedTarget, 1, InputEvent.BUTTON1_DOWN_MASK, MouseEvent.BUTTON1);
         }
```

Comment:
The (BoardView) cast is unnecessary and the public access to gui.bv should probably be removed. If you're in the mood for that :)